package.json

```diff
@@ -150,6 +150,7 @@
     "debug": "^4.1.1",
     "fast-json-patch": "^3.0.0-1",
     "get-stdin": "^6.0.0",
+    "@gitbeaker/node": "21.3.0",
     "gitlab": "^10.0.1",
```

Comment:
```suggestion
```
is gitlab package really still needed when the package moved and you are importing from gitbreaker instead of gitlab?

Reply:
I do step by step. Should be resolved